scripts/install.tpl.sh

```diff
@@ -36,16 +36,16 @@ function kubectl_wait() {
     kubectl wait --for=condition=Available --namespace="${namespace}" "${runtime}" --timeout="${timeout}"
 }
 
-kubectl apply -f "https://github.com/cert-manager/cert-manager/releases/download/${cert_mgr_version}/cert-manager.yaml"
+kubectl apply -f testdata/manifests/cert-manager-${cert_mgr_version}.yaml
```

Comment:
I don't think this works because we re-use this script in the actual github release. So in that case, users installing operator controller will likely need to use the URL like we have it today.